native/android/build.gradle

```diff
@@ -22,6 +22,11 @@ allprojects {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
             url "$rootDir/../node_modules/react-native/android"
         }
+        buildscript {
```

Comment:
this might not be required.

Reply:
is part of the docs https://github.com/wix/detox/blob/master/docs/Introduction.Android.md#setup
so I'll leave it there for now  when it'll be all working I might check if I can remove it, thanks for pointing it out tough 👍